# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FtMoveActionGoal action_goal
FtMoveActionResult action_result
FtMoveActionFeedback action_feedback
